src/WebApp/Startup.cs

```diff
@@ -73,7 +77,29 @@ public void ConfigureServices(IServiceCollection services)
             services
                 .AddRazorPages()
                 .AddRazorRuntimeCompilation()
-                .AddFluentValidation();
+                .AddFluentValidation()
+                .AddViewLocalization(options => options.ResourcesPath = "Resources");
+
+            services.Configure<RequestLocalizationOptions>(
+             opts =>
+            {
+                var settingSupportedCultures = Configuration.GetSection("Localization:Cultures")
+                                                .GetChildren().Select(child => child.Value);
```

Comment:
I can't test this right now, but it should be possible to do this as follows:

~~~csharp
var settingSupportedCultures = Configuration.GetValue<string[]>("Localization:Cultures");
~~~

Reply:
Nope. It was the first thing I tried too.
Making a class is an overkill for now

Reply:
I've tried this now and you are correct, but the following works:

```suggestion
                var settingSupportedCultures = Configuration.GetSection("Localization:Cultures").Get<string[]>();
```

Reply:
Very cool. Couldn't figure it out myself